dev_tool/src/test/web.ts

```diff
@@ -17,17 +16,11 @@ export async function runWebTestsWatch(jestArgs: string[]) {
 
     await installWebDepsOnce(runner)
 
-    // because we are inheriting stdio for this process,
-    // we need to not run spawnSync or else all the output
-    // for the graphql compiler will be swallowed.
-    const proc = spawn('yarn', ['test'].concat(jestArgs), {
```

Comment:
I think this is good because it fixes the crash, but I want to call out that we lose the interactiveness of jest. You can't hit f to re-run failing tests without getting stdio.

Reply:
Oh I didn't notice that. That shouldn't be happening, lerna should support jest running with watch. I peeked at things more and docs are saying our jest setup with lerna may needs improvement. similar to work @mojotalantikite did with eslint, we should have a shared jest config at the root.  Rather than walk down that path I pulled all the tooling changes off this PR can make separate PR next to troubleshoot this stuff.

Reply:
The issue isn't the `lerna` part, it's the `spawn` part. runCommandAndOutput doesn't take input from the terminal, it just runs the process and prints the output. By calling spawn with `stdio: 'inherit'` then it properly accepts terminal input and sends it to the running process. And it's certainly something about that that is causing the kernel panic.

Reply:
I'm very happy with ./dev calling lerna commands, providing a CLI interface with `--help` and the like. The one thing besides that that ./dev is doing that I don't know how to do with lerna is running dependencies correctly. When you run `./dev test web` it's not just calling `yarn test` but also running `lerna run gqlgen:watch`, and `lerna run generate:watch --scope=app-proto`